index.html

```diff
@@ -0,0 +1,30 @@
+<!--HTML layout page-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css" type="text/css">
+    <title>Portfolio | Sully</title>
+</head>
+<body class="main-content">
+    <header class="mainheader">
+        <h2 class="usernamehandle">bettercallsully</h2>
+        <nav class="toolbar">
+            <img class="hamburger" src="myassets/hamburger.png" alt="Hamburger Logo">
+        </nav>
+    </header>
+    <article>

```

Comment:
- [ ] **OPTIONAL** You actually did a great job with [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp).  However, this `<article>` tag isn't following its concepts. Don't you think it's better if you define this part of the page as a `<section>` tag only? This way you'll be following the Semantic HTML concepts by specifying the sections in a `<section>` tag and when there's an article within the website, then you can use the `<article> tag. Please tell me what do you think of that!